librz/core/cmd/cmd_help.c

```diff
@@ -130,6 +130,70 @@ RZ_IPI RzCmdStatus rz_cmd_help_search_handler(RzCore *core, int argc, const char
 	return status;
 }
 
+static char *cons_hud_help_string(const char *s) {
+	if (!rz_cons_is_interactive()) {
+		eprintf("Hud mode requires scr.interactive=true.\n");
+		return NULL;
+	}
+	char *os, *track, *ret, *o = rz_str_dup(s);
```

Comment:
Please give `o` and `os` more descriptive names. It is a little hard to follow what they are for.

Reply:
Ok 5efafe549b01a0286d4295bcfa2d23f1eaae38ef. Sorry original naming was from `rz_cons_hud_string()`.